packages/eslint-config-eslint/default.yml

```diff
@@ -87,7 +86,6 @@ rules:
     no-path-concat: "error"
```

Comment:
Can we add `no-console` since it was removed?

Reply:
It was at https://github.com/eslint/eslint/pull/11518/files#diff-5d1f0612c1d7040125f0e4e9539c9eb8L57.

seems it has been  inconsistency here --- some recommended rules were included, while some were not.